src/handbook/company/security/asset-management.md

```diff
@@ -0,0 +1,79 @@
+---
+navTitle: Asset Management Policy
+---
+
+# Access Control Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @knolleary     | 2023-06-01     |
+
+## Purpose
+
+To identify organizational assets and define appropriate protection responsibilities.
+To ensure that information receives an appropriate level of protection in accordance
+with its importance to the organization. To prevent unauthorized disclosure,
+modification, removal, or destruction of information stored on media.
+
+
+## Scope 
+
+This policy applies to all FlowForge owned or managed information systems.
+
+## Policy
+
+### Inventory of Assets
+
+Assets associated with information and information processing facilities that
+store, process, or transmit classified information shall be identified and an
+inventory of these assets shall be created and maintained.
+
+### Ownership of Assets
+
+Assets maintained in the inventory shall be owned by a specific individual or
+group within the company.
+
+### Acceptable Use of Assets
+
+Rules for the acceptable use of information, assets, and information processing
+facilities shall be identified and documented in the [Information Security Policy](./information-security.md).
+
+### Loss or Theft of Assets
+
+All personnel must immediately report the loss of any information systems,
+including portable or laptop computers, smartphones, PDAs, authentication tokens
+(keyfobs, one-time-password generators, or personally owned smartphones or
+devices with access to FlowForge systems) or other devices that can store and
+process or help grant access to FlowForge data.
+
+### Return of Assets
+
+All employees and third-party users of FlowForge equipment shall return all of
+the organizational assets within their possession upon termination of their
```

Comment:
Please link to this doc from the peopleops off boarding section.

Reply:
Done